[2024-12-27T01:51:28.838+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-12-27T01:51:26.349470+00:00 [queued]>
[2024-12-27T01:51:28.842+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-12-27T01:51:26.349470+00:00 [queued]>
[2024-12-27T01:51:28.842+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-12-27T01:51:28.842+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-12-27T01:51:28.842+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-12-27T01:51:28.847+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-12-27 01:51:26.349470+00:00
[2024-12-27T01:51:28.850+0000] {standard_task_runner.py:55} INFO - Started process 7637 to run task
[2024-12-27T01:51:28.852+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2024-12-27T01:51:26.349470+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp8y9lbe3n']
[2024-12-27T01:51:28.854+0000] {standard_task_runner.py:83} INFO - Job 19: Subtask is_api_available
[2024-12-27T01:51:28.879+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2024-12-27T01:51:26.349470+00:00 [running]> on host f750dc751645
[2024-12-27T01:51:28.905+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-12-27T01:51:26.349470+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-27T01:51:26.349470+00:00
[2024-12-27T01:51:28.906+0000] {http.py:120} INFO - Poking: api/
[2024-12-27T01:51:28.909+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-12-27T01:51:28.910+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-12-27T01:51:33.581+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-12-27T01:51:33.586+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20241227T015126, start_date=20241227T015128, end_date=20241227T015133
[2024-12-27T01:51:33.617+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-12-27T01:51:33.635+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
